src/documents/views/NewDocumentView.js

```diff
@@ -91,7 +91,7 @@ const NewDocumentView = ({match, history, location}) => {
             fileNode={location.state.file}
             handleSubmit={handleSubmit}
             history={history}
-            showFieldHints={true}
+            showFieldHints={false}
```

Comment:
we still want to show the `"Please provide a descriptive title without dates or adjectives such as "new", "updated", "final", etc."` message as well as the `EditExistingDocs` notification, setting this to false blocks both of these